WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -242,21 +224,13 @@ import WordPressShared
         return message
     }
     
-    func noResultsButtonText() -> String? {
-        if syncHelper?.isSyncing == true || isSearching() {
+    override func noResultsButtonText() -> String? {
+        if syncHelper.isSyncing == true || isSearching() {
             return nil
         }
         
         let filter = currentPostListFilter()
-        
-        // currentPostListFilter() may return `nil` at this time (ie: it's been declared as
-        // `nullable`).  This will probably change once we can migrate
-        // AbstractPostListViewController to Swift, but for the time being we're defining a default
-        // filter here.
-        //
-        // Diego Rey Mendez - 2016/04/18
-        //
-        let filterType = filter?.filterType ?? .Draft
+        let filterType = filter.filterType
```

Comment:
@diegoreymendez same here if you want to match `PageListViewController.swift Line:169` as `let filterType = currentPostListFilter().filterType`. (no big deal)